index.js

```diff
@@ -49,9 +49,12 @@ class ChartistGraph extends Component {
   }
 
   render() {
-    const className = this.props.className ? ' ' + this.props.className : ''
-    const style = this.props.style ? this.props.style : {};
-    return (<div className={'ct-chart' + className} ref='chart' style={style} />)
+    const { className, style, children, data, type } = this.props;
+    return (
+      <div className={`ct-chart ${className || ''}`} ref='chart' style={style}>
+         {children && cloneElement(children, { type, data })}
```

Comment:
I only used `cloneElement` method once when I have to passing props down to children with `react-router`, I've learned from others example it might be better to do a `map` function over the children in case user passing more than one child. 

```js
const children = React.Children.map(this.props.children, child => (
  React.cloneElement(child, {
    type,
    data
  })
))

return (
  <div>{children}</div>
)
```